src/Illuminate/Queue/Console/stubs/jobs.stub

```diff
@@ -17,7 +17,7 @@ class Create{{tableClassName}}Table extends Migration
             $table->bigIncrements('id');
             $table->string('queue');
             $table->longText('payload');
-            $table->tinyInteger('attempts')->unsigned();
+            $table->unsignedTinyInteger('attempts');
             $table->unsignedInteger('reserved_at')->nullable();
             $table->unsignedInteger('available_at');
             $table->unsignedInteger('created_at');
```

Comment:
is it normal that it is unsignedInteger instead of timestamps for reserved_at, available_at and created_at ?